### DISCUSSION TITLE
Chrome defaulting to HTTPS

### DISCUSSION THREAD

**Discussion by terrorist96** [EFForg/https-everywhere#19898] [Q&A] [answered]
[//]: # (Thank you for reporting an issue to HTTPS Everywhere.)
[//]: # (We welcome input from users on improving this project.)
[//]: #
[//]: # (Please help us by following this issue template.)
[//]: # (You can delete all blank lines and all lines starting)
[//]: # (with the comment marker, such as this one.)

Type: other

[//]: # (Include any other relevant information below. Thank you again for)
[//]: # (helping to improve HTTPS Everywhere.)
Chrome will soon start defaulting to HTTPS. See:

https://bugs.chromium.org/p/chromium/issues/detail?id=1141691
https://chromium-review.googlesource.com/c/chromium/src/+/2620462
https://9to5google.com/2021/01/11/google-chrome-address-bar-may-soon-default-to-https/

What will this mean for HTTPS Everywhere? Will the extension soon become superfluous?

**Comment by zoracon** [comment, marked as answer]
So this is very exciting and good news! I will watch this flag. The only downside is that they still redirect to HTTP after trying HTTPS, which is not ideal. Hopefully they will add an option or interstitial that allows the user to decide if they want to navigate to the HTTP page or not.
